jwst/master_background/master_background_step.py

```diff
@@ -73,7 +78,25 @@ def process(self, input):
                 self.record_step_status(result, 'master_background', success=False)
 
                 return result
+            # Check if subtract_background is set to False -> skip step
+            if self.subtract_background is False:
```

Comment:
I think you first need to check whether `self.subtract_background` is None and if it isn't, then you can check for False. If it's None I think the current line will fail (because it doesn't exist).

Reply:
To elaborate a little on what @hbushouse wrote, this line should be:

    if self.subtract_background is not None and not self.subtract_background:

The point is that you're not supposed to check boolean variables by `if x is True:` or `if x == True:` (or False); you're supposed to use this syntax:  `if x:` or `if not x`.  This is a PEP 8 issue.